lightwood/encoder/image/img_2_vec.py

```diff
@@ -9,12 +9,25 @@
 
 
 class Img2VecEncoder(BaseEncoder):
+    """
+    This encoder generates encoded representations for images using a pre-trained deep neural network.
+
+    All input images are rescaled to a standard size of 224x224, and normalized using the mean and standard deviation of the ImageNet dataset (as it was used to train the underlying NN).
```

Comment:
im ok with the size but is there a reason we picked 224 x 224? could this be a kwarg?

Reply:
Because it's the standard for torchvision pretrained models. We could turn it into a kwarg, but then we'd have to rewrite this to be trainable from scratch with different architectures, which I'd rather not do here.